### DISCUSSION TITLE
Broken Images - Pimcore 6.8.4

### DISCUSSION THREAD

**Discussion by alinanicoletaroman** [pimcore/pimcore#8155] [Q&A / Support] [answered]
Some images are not correctly displayed in Pimcore Version 6.8.4. Some of them have transparent background or the image is completly broken and some of them have no preview: 

![image](https://user-images.githubusercontent.com/46678202/107679461-e0f85080-6c9c-11eb-874f-70c64eb4c128.png)
![image](https://user-images.githubusercontent.com/46678202/107679506-ef466c80-6c9c-11eb-89d9-c5f474e3c04c.png)

If i download them, they are correclty displayed. 

imagick module version | 3.4.4
Imagick compiled with ImageMagick version | ImageMagick 6.8.9-9 Q16 x86_64 2020-12-04 http://www.imagemagick.org
PHP 7.4

and the following extensions installed: 
![image](https://user-images.githubusercontent.com/46678202/107679773-43e9e780-6c9d-11eb-85f0-ceb67db7d097.png)

Image example: 
![122397-8](https://user-images.githubusercontent.com/46678202/107679837-59f7a800-6c9d-11eb-8588-416d8f6a019a.jpg)

Have you any idea how to fix this issue?
Thanks in advance!

**Comment by dvesh3** [comment, marked as answer]
Images are clipped automatically, if they have clipping path. If you don't want auto clipping then you can disable it with config:
```yaml
pimcore:
    assets:
        image:
            thumbnails:
                clip_auto_support: false
```
Please make sure to clear temporary files after configuring.

ref: https://pimcore.com/docs/pimcore/current/Development_Documentation/Assets/Working_with_Thumbnails/Image_Thumbnails.html#page_Clipping-Support

**Reply by alinanicoletaroman** [reply]
Many thanks!!! 🎉
